app/test/test-support/src/main/java/io/syndesis/test/container/dockerfile/SyndesisDockerfileBuilder.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
```

Comment:
2016?

Reply:
I think all of our files are using that date and it is protected by an enforcer rule. I once got build errors while using another date here.

Reply:
ok